mets_adapter/mets_adapter/src/main/scala/uk/ac/wellcome/mets_adapter/services/MetsStore.scala

```diff
@@ -0,0 +1,35 @@
+package uk.ac.wellcome.mets_adapter.services
+
+import uk.ac.wellcome.mets_adapter.models._
+import uk.ac.wellcome.storage.store.VersionedStore
+import uk.ac.wellcome.storage.{Identified, NoVersionExistsError}
+
+/** Stores METS data. The data should first be filtered to ensure storing is
+  *  performed only if it is a newer version and the location is pointing to a
+  *  different XML file.
+  */
+class MetsStore(store: VersionedStore[String, Int, MetsData]) {
```

Comment:
Stupid question: is this a full VHS or just dynamo?

Reply:
Assumed was just a dynamo, as what we are storing is pretty small at the moment (the `MetsData` case class).

Ideally the types would be set up in such a way so we could pass a VHS in here as the argument if we wanted, with the `MetsStore` being agnostic to the exact implementation details of whether VHS or not, but the VHS currently would require a`VersionedStore[String, Int, HybridStoreEntry[MetsData, EmptyMetadata]]`, so that wouldn't be possible at the moment.